R/extract.R

```diff
@@ -25,6 +25,10 @@
 #'
 #' - `extract_workflow()` returns a workflow, possibly fit.
 #'
+#' - `extract_dials_parameter()` returns a single dials parameter object.
+#'
+#' - `extract_dials_parameter_set()` returns a set of dials parameter objects.
```

Comment:
Backwards naming? 😆

Reply:
aah :see_no_evil: thanks for catching that!